src/cace/CMakeLists.txt

```diff
@@ -130,9 +130,14 @@ target_include_directories(
   cace PUBLIC
   $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>
 )
+
+get_filename_component(CACE_BINARY_PARENT "${CMAKE_CURRENT_BINARY_DIR}" DIRECTORY)
+
+target_include_directories(cace PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>)
```

Comment:
I don't think this is correct. Any users including CACE headers should name them as 
```c
#include <cace/asdf.h>
```